### PR TITLE
super clown shoes

### PR DESCRIPTION
**Pull request by unid15**
adminbus only

because clown totally needs more ways to become valid

### CODE REVIEW

**Comment by PJB3005** [comment]
:+1: MERGE WHEN

Make it gib them if they're a mime :^)

**Comment by FudgePucker** [comment]
>Laughtrack

Oh god now there can be a live studio audience for security chasing the clown.

**Comment by dylanstrategie** [comment]
>Span class in caps

**Comment by CptWad** [comment]
Thank you honkmother :pray:

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 1]
```diff
@@ -135,11 +137,99 @@
...
+	var/self_loc = src.loc //To disallow fucking with shoes from distance
+	var/new_sound = input(user,"Select the new step sound!","Advanced clown shoes") in sound_list
+
+	if(user_loc==user.loc && self_loc==src.loc)
+		step_sound = sound_list[new_sound]
+		user << "<span class='sinister'>You set the step sound to \"[new_sound]\"!</span>"
+	else
+		user << "<span class='warning'>You have to stand still!</span>"
+
+/obj/item/device/soundsynth/verb/ChangeSound()
```
Comment (ComicIronic) [inline code/modules/clothing/shoes/miscellaneous.dm]: What

Comment (ComicIronic) [inline code/modules/clothing/shoes/miscellaneous.dm]: You actually have to fix this.

Comment (PJB3005) [inline code/modules/clothing/shoes/miscellaneous.dm]: From unfit's shuttle PR: 
>On 4th of July I'm going on a holiday and I won't be able to work on this for 2 weeks

Comment (ComicIronic) [inline code/modules/clothing/shoes/miscellaneous.dm]: For the love of fugg

Comment (Unfit) [inline code/modules/clothing/shoes/miscellaneous.dm]: I can't fix it as I'm using a phone atm, but if somebody were so nice as to make a PR to my repo fixing that typo, I could merge it
If not, well, I'd be back in a week

Comment (PJB3005) [inline code/modules/clothing/shoes/miscellaneous.dm]: https://github.com/Unfit/vgstation13/pull/2

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 2]
```diff
@@ -135,11 +137,99 @@
...
+	set name = "Change Sound"
+
+	return src.attack_self(usr)
+
+/obj/item/clothing/shoes/clown_shoes/advanced/step_action()
+	if(ishuman(loc))
+		var/mob/living/carbon/human/H = loc
+
+		if(H.mind && H.mind.assigned_role != "Clown")
+			if(prob(1) || ( H.mind.assigned_role == "Mime" && prob(10) ) )
```
Comment (d3athrow) [inline code/modules/clothing/shoes/miscellaneous.dm]: >prob and rand not once but 3 times per step

what the fuck no

**Comment by Unfit** [comment]
okay should be less bad now

added a fart sound and a random sound option (which when turned on selects a random sound on every step)

**Pull request merged**